app-editors/zed/zed-0.157.5.ebuild

```diff
@@ -47,7 +47,8 @@ LICENSE+="
 	LGPL-3 MIT MPL-2.0 Unicode-DFS-2016 ZLIB
 "
 SLOT="0"
-KEYWORDS="~amd64"
+KEYWORDS="~amd64 ~arm64"
+IUSE="gles"
 CHECKREQS_DISK_BUILD="8G"
```

Comment:
Btw, with `FEATURES="test"`:
` * Final size of build directory: 9615476 KiB (  9.1 GiB)`